.travis.yml

```diff
@@ -22,7 +22,7 @@ env:
   global:
     - BUILD_ID=${TRAVIS_BUILD_ID}
     - AIRFLOW_CONTAINER_MOUNT_HOST_VOLUMES="false"
-    - AIRFLOW_CI_SILENT="false"
+    - AIRFLOW_CI_SILENT="true"
```

Comment:
Would not that make debugging failures difficult?

Reply:
It's mostly for diagnostic messages that were useful at the beginning to see if everything is fine. It's always the question of noise vs. debuggability :(. 

I think I wanted to see the output and decide if it is good to merge this one. I think also when we move to GitLab we will be able to send the verbose output to files and store them as artifacts in GitLab. It's all prepared to do so.

Reply:
@kaxil -> take a look at the output. It's fine. We are still using `set +x` in the actual tests so we have very detailed information but in static check SILENT flag will make it much nicer to see the result of static checks directly. It looks good for me now to merge it.

Reply:
Yeah. It gets really clumsy without it (we see all files that are checked) so we better merge it.

Reply:
👍